corehq/ex-submodules/dimagi/utils/couch/sync_docs.py

```diff
@@ -45,6 +45,8 @@ def index_design_docs(db, docid, design_name, wait=True):
             except HTTPError as e:
                 if 'timeout' not in six.text_type(e) and e.response.status_code != 504:
                     raise
+                elif abort_on_timeout:
```

Comment:
why quit on timeout for auditcare?

Reply:
I think since a request to the auditcare view will time out, it was just pinging that view in a loop. Instead of aggressively making sure that the view is up to date before exiting, pinging multiple times if necessary, I want it to ping and wait but then just exit if it times out. Couch will still be processing the view at that point and doesn't need to be repinged.